### PR TITLE
version 1.1.3 🎉

### PR DESCRIPTION
**Pull request by ritiksr25**
- email service: updates and improvements
- sending general emails: from full html
- multithreaded
- Readme and package.json details updated

### CODE REVIEW

File: config/emailService.js [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
 const transporter = nodemailer.createTransport({
-	service: "gmail",
 	type: "SMTP",
-	host: "smtp.gmail.com",
+	host: EMAIL_HOST,
+	secure: true,
+	debug: true,
+	port: 465,
 	auth: {
 		user: EMAIL_USER,
```
Comment (rhnmht30) [inline config/emailService.js:15]: since our email service is replaced, is this auth parameter required?

File: controllers/events_controller.js [hunk 2]
```diff
@@ -1260,42 +1233,33 @@ module.exports.generateCerti = async (req, res) => {
...
-
-	let event;
-	let systemMailTypes = ["event-reminder", "event-followup", "event-thanks"];
-	if (!subject && !content && systemMailTypes.indexOf(type) !== -1) {
-		event = await Event.findById(eid);
-		if (!event) {
-			return sendError(res, "Invalid Event!!", BAD_REQUEST);
-		}
-	}
+	let { users, subject, content, type } = req.body;
```
Comment (rhnmht30) [inline controllers/events_controller.js:1236]: is `content` here a raw HTML file ?

Comment (ritiksr25) [reply controllers/events_controller.js:1236]: Yes

File: config/emailService.js (hunk 1 above)
Comment (ritiksr25) [reply config/emailService.js:15]: Yes, any email service require auth

**Pull request merged**